articles/operator-service-manager/best-practices-onboard-deploy.md

```diff
@@ -8,164 +8,173 @@ ms.topic: best-practice
 ms.service: azure-operator-service-manager
 ---
 
-# Azure Operator Service Manager Best Practices to onboard and deploy a Network Function (NF)
+# Azure Operator Service Manager Best Practices to onboard and deploy Network Functions
 
 Microsoft has developed many proven practices for managing Network Functions (NFs) using Azure Operator Service Manager (AOSM). This article provides guidelines that NF vendors, telco operators and their System Integrators (SIs) can follow to optimize the design. Keep these practices in mind when onboarding and deploying your NFs.
 
-## Technical overview
+## General considerations
 
-- Onboard an MVP first.
+We recommend that you first onboard and deploy your simplest NFs (one or two charts) using the quick starts to familiarize yourself with the overall flow. Necessary configuration details can be added in subsequent iterations. As you go through the quick starts, consider the following:
 
-- You can add config detail in subsequent versions.
+- Structure your artifacts to align with planned use. Consider separating global artifacts from those you want to vary by site or instance.
+- Ensure service composition of multiple NFs with a set of parameters that matches the needs of your network. For example, if your chart has a thousand values and you only customize a hundred of them, make sure in the AOSM CGS layer (covered more extensively below) you only expose that hundred.
```

Comment:
I don't believe we are allowed to use AOSM as an acronym. It must be expanded with AOSM in ( ), like this: Azure Operator Service Manager (AOSM)